assets/css/addon.css

```diff
@@ -488,3 +488,30 @@ lead {
   font-weight: 600;
   color: var(--primary-button-foreground-color) !important;
 }
+
+.profile {
+  float: left;
+  border: 1px solid black;
```

Comment:
Use var(--primary-region-border-color) instead of black. I would also set var(--primary-region-border-radius) for border-radius here

Reply:
Done.